src/ORM/Filters/SearchFilter.php

```diff
@@ -26,7 +29,19 @@
  */
 abstract class SearchFilter
 {
-    use Injectable;
+    use Injectable, Configurable;
+
+    /**
+     * Whether the database uses case sensitive collation or not.
+     * @internal
+     */
+    private static ?bool $caseSensitiveByCollation = null;
```

Comment:
```suggestion
    private static ?bool $case_sensitive_by_collation = null;
```

Reply:
This isn't a config property, so we should use camelcase.

Reply:
Ah ok, I missed that bit